lib-ee/emqx_license/etc/emqx_license.conf

```diff
@@ -1,5 +1,4 @@
 license {
-    type = key
     # The default license has 1000 connections limit, it is issued on 20220419 and valid for 5 years (1825 days)
```

Comment:
move this to i18n

also: delete the 'type' description in i18n

Reply:
I think I deleted the type description in i18n? https://github.com/emqx/emqx/pull/8961/files#diff-a42874a345b13646d792ddcffaae95be3972e37b4d850358d9a6f85c827d6b61L23

Did it slip past somewhere else?  I couldn't find mentions of type there :see_no_evil:

Reply:
Sorry, I missed it.

Please move this comment to the `key_field` description in i18n.

Reply:
:heavy_check_mark: